apps/omg_db/lib/omg_db/rocks_db.ex

```diff
@@ -11,143 +11,142 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-if Code.ensure_loaded?(:rocksdb) do
```

Comment:
the diff is garbled here. What did here change is only the unindent and removal of the `if`?

Same question for `...core.ex` and `..server.ex` files

Reply:
yes makes sense! I've now added the ensure loaded calls back and will remove them in a separate PR.

Reply:
Ah, I was just asking if you only did the unindent.

If only the unindent, then all :+1: and no objections to getting rid of the `if` right away.